src/remoteplayer.cpp

```diff
@@ -115,37 +115,46 @@ void RemotePlayer::save(std::string savedir, IGameDef *gamedef)
 	return;
 }
 
-void RemotePlayer::deSerialize(std::istream &is, const std::string &playername)
+void RemotePlayer::deSerialize(std::istream &is, const std::string &playername,
```

Comment:
same problem here. RemotePlayer & SAO are not linked at this moment, it's too early in the process